src/ms-store-badge.js

```diff
@@ -27,6 +27,18 @@ class MSStoreBadge extends HTMLElement {
          * The ID of your app.
          */
         this.productId = "";
+        /**
```

Comment:
Can you delete this file, commit, then add this file to the .gitignore list? Generated files don't need to be in source control.